scripts/presubmit.sh

```diff
@@ -12,7 +12,7 @@ find . -name "mock_*" -delete
 golint --set_exit_status ./...
 go vet ./...
 misspell -error -locale US .
-nolicense=$(find . -name "*.go" | xargs grep -L "Apache License")
+nolicense=$(find . -path ./protoc -prune -name "*.go" -o -name "*.proto" | xargs grep -L "Apache License")
```

Comment:
If I run this locally I don't get any ".go" files in the output.

The best I got was `find . -name '*.go' -or -name '*.proto' | grep -v ./protoc`.

(Btw, nit: single quotes for '*.go' and '*.proto')